drivers/hmc5883l/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = hmc5883l
```

Comment:
no need to specify this line, it is deducted automatically by the make system

Reply:
A number of existing drivers are defining it in same way. So I thought that it is necessary.

Reply:
Fixed